torch/autograd/__init__.py

```diff
@@ -179,5 +179,17 @@ def variable(*args, **kwargs):
     warnings.warn("torch.autograd.variable(...) is deprecated, use torch.tensor(...) instead")
     return torch.tensor(*args, **kwargs)
 
+# [Experimental] Configure number of threads on each device to concurrently run autograd backward.
```

Comment:
I think you want to add these as proper docstrings and add the corresponding `.. autofunction:: get_num_threads_per_device ` to `docs/source/autograd.rst`.
See the `.grad` function above for an example.